blockchain/beacon/src/drand.rs

```diff
@@ -201,15 +201,18 @@ impl DrandBeacon {
         let chain_info = &config.chain_info;
 
         if cfg!(debug_assertions) && config.network_type == DrandNetwork::Mainnet {
-            let client = https_client();
-            let remote_chain_info: ChainInfo = client
-                .get(format!("{}/info", &config.server).try_into()?)
-                .await?
-                .into_body()
-                .json()
-                .await
-                .map_err(|e| anyhow::anyhow!("{}", e))?;
-            debug_assert!(&remote_chain_info == chain_info);
+            futures::executor::block_on(async {
```

Comment:
How about using [`tokio::runtime::Handle::current().block_on`](https://docs.rs/tokio/latest/tokio/runtime/struct.Handle.html#method.block_on)

Reply:
I think `futures::executor` is more clear and less tied to the brittle `tokio` interface. We have `futures` in our dependencies anyway. Is it fine for you?

Reply:
That's fine if it works, I'm just worried that it will throw errors if "tokio::spawn" is called inside the async function